StructureDefinition/StructureDefinition-ror-accessibility-location.json

```diff
@@ -0,0 +1,50 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "ror-accessibility-location",
+  "url": "http://esante.gouv.fr/fhir/ror/StructureDefinition/ror-accessibility-location",
+  "version": "2.4",
```

Comment:
Pourquoi les SearchParam sont en v1 et les profils en v2.4 ?

Reply:
The valueSet reference https://test on element Extension.value[x] could not be resolved